sourcegraph/sourcegraph.proto

```diff
@@ -102,6 +102,10 @@ message Repo {
 	// URL. E.g., "github.com/user/repo".
 	string uri = 1 [(gogoproto.customname) = "URI"];
 
+	// Origin is populated for repos fetched via federation or
+	// discovery. It is the hostname of the host that owns the repo.
+	string origin = 21;
```

Comment:
In general no, but since this is prerelease we can be pretty sure nobody else is relying on the field tags. We aren't (we are persisting the struct in the DB but not as protobuf blobs).

Sent from my iPhone

> On Jun 20, 2015, at 12:47, Dmitri Shuralyov <notifications@github.com> wrote:
> 
> In sourcegraph/sourcegraph.proto:
> 
> > @@ -102,6 +102,10 @@ message Repo {
> >  	// URL. E.g., "github.com/user/repo".
> >  	string uri = 1 [(gogoproto.customname) = "URI"];
> >  
> > +	// Origin is populated for repos fetched via federation or
> > +	// discovery. It is the hostname of the host that owns the repo.
> > +	string origin = 21;
> Is it okay to reuse a previously used but now deleted id 21?
> 
> —
> Reply to this email directly or view it on GitHub.
>